config/config.properties

```diff
@@ -201,14 +201,17 @@ metrics.enableStatistics=false
 #metrics.influxDBURI=http://localhost:8086
 
 # InfluxDB database name.
-#metrics.influxDBName=pravegaInflux
+#metrics.influxDBName=pravega
 
 # InfluxDB user name.
 #metrics.influxDBUserName=
 
 # InfluxDB password.
 #metrics.influxDBPassword=
 
+# InfluxDB retention policy (e.g. 2h, 52w).
+#metrics.influxDBRetention=
```

Comment:
Would it be interesting to set a safe default here?

Reply:
Here the retention policy name is arbitrary, similar to influxDBUserName, no default value for it.

Reply:
The default in InfluxDB is no user (authentication is not enabled) and no password.

As to the retention policy, InfluxDB databases have `default` retention policies (name of the default retention policy is set per database, by InfluxDB), so if it not specified here, then the default one will be used.